Makefile

```diff
@@ -38,6 +38,7 @@ eunit:
 	${REBAR} as test eunit
 
 ct:
+	openssl rand -out test/partisan_SUITE_data/RAND 4096
```

Comment:
If this becomes a problem in the future, one could commit the `RAND` file to the repository, as OTP does.